src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__description {
+    color: $main-text-color;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    height: 36px;
+    width: 166px;
+  }
+
+  &__code {
+    color: $secundary-text-color;
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    height: 16px;
```

Comment:
Same issue here. Avoid setting the height manually and let the content define the height of the element.